src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -274,7 +279,12 @@ method messageAdded*(self: Module, message: MessageDto) =
   let index = self.view.model().findIndexForMessageId(message.replace)
   if(index != -1):
     self.view.model().removeItem(message.replace)
-
+  
+  # Must be fixed in future
```

Comment:
It would be cool if you elaborated what `Must be fixed in the future` means.
Judging by the code, you're aiming to not process deleted messages (okay cool) but what must be fixed and why?

Reply:
@PascalPrecht as I understand, we can show for some time kind of `undo` for deleted messages.

Reply:
Alright, please add that to the comments, otherwise we can remove the comments entirely

Reply:
You can link the github issue @borismelnik https://github.com/status-im/status-desktop/issues/7632